openverse_catalog/dags/common/slack.py

```diff
@@ -252,17 +271,41 @@ def should_send_message(http_conn_id=SLACK_NOTIFICATIONS_CONN_ID):
     except AirflowNotFoundException:
         return False
 
+    # Exit early if this DAG is configured to skip Slack messaging
+    if should_silence_message(text, username, dag_id):
+        log.info(f"Skipping Slack notification for {dag_id}.")
+        return False
+
     # Exit early if we aren't on production or if force alert is not set
     environment = Variable.get("environment", default_var="dev")
     force_message = Variable.get(
-        "slack_message_override", default_var=False, deserialize_json=True
+        "slack_message_override", default_var={}, deserialize_json=True
     )
     return environment == "prod" or force_message
 
 
+def should_silence_message(text, username, dag_id):
```

Comment:
Do you mind moving this function to above where it is used within the module?

Reply:
Totally! Also moved `send_message` for the same reason.